### ISSUE TITLE
Correlating an assembly between CLR MD and reflection

### ISSUE THREAD

**Issue by PathogenDavid** [microsoft/clrmd#284]
Is there a way to correlate a `System.Reflection.Assembly` instance in the target process with a `Microsoft.Diagnostics.Runtime.ClrModule` in CLR MD?

Alternatively:

* Is there a way to get the [`ModuleVersionId`](https://docs.microsoft.com/en-us/dotnet/api/system.reflection.module.moduleversionid) from CLR MD?
* Is there a way to get the `ClrModule.AssemblyId` of an arbitrary `System.Reflection.Assembly`?

Basically I am wanting to have a .NET app request a separate CLR MD app to attach to it and report back information about types from a specific assembly.

**Comment by leculver** [comment]
Unfortunately I'm not sure.  You would have to dig into the .Net Core implementation or field values to know, but I don't think there's a way to do that.

**Comment by PathogenDavid** [comment]
Sorry to hear that. Thanks for the response!
